src/components/select/select.component.spec.ts

```diff
@@ -43,6 +43,10 @@ describe('Select Component', () => {
         fixture.detectChanges();
     });
 
+    it('should be initially blank if no placeholder is set', () => {
+        expect(component.input).toBe('');
```

Comment:
I think perhaps we should be testing the element `placeholder` property instead of the `input` property, perhaps testing both when empty and when populated? I don't believe the `input` value would ever change when the placeholder value changes?